boards/nxp/imx943_evk/imx943_evk_mimx94398_m33.yaml

```diff
@@ -17,4 +17,5 @@ supported:
   - uart
   - netif:eth
   - gpio
+  - flexio
```

Comment:
I think it should be "pwm" but not flexio

Reply:
sure,np.